dev-libs/tomsfastmath/files/tomsfastmath-0.13.1-makefile-fix.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/makefile.shared b/makefile.shared
+index 47c35d9..5894688 100644
+--- a/makefile.shared
++++ b/makefile.shared
+@@ -96,6 +96,9 @@ install: $(LIBNAME)
+ 	libtool --mode=install install -c $(LIBNAME) $(DESTDIR)$(LIBPATH)/$(LIBNAME)
+ 	install -d -g $(GROUP) -o $(USER) $(DESTDIR)$(INCPATH)
+ 	install -g $(GROUP) -o $(USER) $(HEADERS_PUB) $(DESTDIR)$(INCPATH)
++	sed -e 's,^Version:.*,Version: $(VERSION),' tomsfastmath.pc.in > tomsfastmath.pc
++	install -d $(LIBPATH)/pkgconfig
```

Comment:
I think these two `install` commands should have a `$(DESTDIR)` prefix, as they do in e.g. https://github.com/libtom/tomsfastmath/blob/develop/makefile.shared#L61

Reply:
You're right. They made a lot of changes to the Makefile(s) -- I noticed a bunch of them and figured I'd just grab the pkgconfig goodies, but combined with your other comments about what param are needed + I should use, this makes sense to do.